Added dataset Liu2012 on aluminium recycling.

stefanpauliuk opened the pull request.

Commit:
Added dataset Liu2012 on aluminium recycling.

nheeren commented:
Nice one - thank you ⭐️ 
A few comments:
1. In the csv file, I will replace `;` with `,` for overall consistency. 
2. Also I will start working on a guideline document that defines these kind of questions. Any objections?
3. Since you used WPD, would you also be able to provide its raw (json) files? All the other data contributions have this so far. This would help others to verify your measurement in the future. 
4. I am undecided if the above issue should be a requirement in this repository. I am inclined to say no, as we, first of all, care about the data. Any opinion?

stefanpauliuk commented:
1) Excel just uses ; when exporting csv, but consistency matters more.
3 and 4) I did not save anything, no json file is available. To me, WPD is an emergency tool and not something that we should use by default. Authors should always be contacted first. 

I agree that intermediate files are important for traceability, but at what point does the whole procedure turn into overkill? In the example of the figure I extracted data from, some lines and data points were not visible due to overlap. Just quickly fudged in some intermediate points and made a comment in the .md file. But that is not an accurate solution. Again, best way is to just ask for data behind figure.

nheeren commented:
Dear Stefan. Good points. I will try to document this further. One idea I had, based on our exchange, is that we could have an optional quality indicator, similar to the Pedigree matrix in ecoinvent.

Commit:
Update Liu2012_FigS7.csv

Replaced `;` with `,`, removed whitespaces, and removed empty lines at the end.
